src/ert/run_models/run_arguments.py

```diff
@@ -42,6 +44,7 @@ def __post_init__(self) -> None:
 class EvaluateEnsembleRunArguments(SimulationArguments):
     active_realizations: List[bool]
     current_case: str
+    analysis_module: str = "Evaluate ensemble"
```

Comment:
We are just depending on this being in simulation arguments, right? So if we want to refactor this we just need to keep this? The reason I am asking is because we might want to remove `current_case` for example, and would like that to be easy to do.